appcues/src/main/java/com/appcues/AppcuesConfig.kt

```diff
@@ -100,4 +100,6 @@ public data class AppcuesConfig internal constructor(
      * a different package than your main app (multi-module apps)
      */
     var packageNames: List<String> = arrayListOf()
+
+    internal var isTesting: Boolean = false
```

Comment:
wondering if naming like `isSnapshotTesting` would make the usage of this even more clear